.github/workflows/pipeline.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
         - name: Install Python 3
           uses: actions/setup-python@v1
           with:
-            python-version: 3.10.2
+            python-version: 3.10.4
```

Comment:
Anything to change with python version for unit testing has a really really low risk of changing anything in NKT.
So this change shouldn't be an issue.